workstation-external-ip.tf

```diff
@@ -0,0 +1,18 @@
+#
+# Workstation External IP
+#
+# This configuration is not required and is
+# only provided as an example to easily fetch
+# the external IP of your local workstation to
+# configure inbound EC2 Security Group access
+# to the Kubernetes cluster.
+#
+
+data "http" "workstation-external-ip" {
+  url = "http://ipv4.icanhazip.com"
```

Comment:
Just a word of caution - when working on engagements I'd refrain from relying on an "untrusted" source like this.  There's nothing saying that they're actually returning _your_ IP address 😄 .  It's fine for this case though.